Sprint-3/implement/get-angle-type.js

```diff
@@ -1,27 +1,24 @@
-// Implement a function getAngleType
+function getAngleType(angle) {
```

Comment:
Nice! Is there a way you could refactor this to make it simpler and clearer? Do you need all these else statements?

Reply:
Yes this how can it be done:
function getAngleType(angle) {
    if (angle < 90) return "acute";
    if (angle === 90) return "right";
    if (angle < 180) return "obtuse";
    return "straight";
}
This version removes unnecessary else statements for clarity and conciseness.